auth_supplier/security/auth_supplier_security.xml

```diff
@@ -4,7 +4,6 @@
     <record id="group_auth_supplier" model="res.groups">
         <field name="name">Supplier Portal</field>
         <field name="category_id" ref="base.module_category_extra"/>
-        <field name="is_portal" eval="True"/>
```

Comment:
is generates a warning because the module doesn't depend on `portal`. Looking at the code, it seems it doesn't need to depend on portal, which would be the other fix. Or am I missing something?